packages/compass-e2e-tests/helpers/compass.js

```diff
@@ -365,6 +398,8 @@ async function beforeTests(unlockKeychain = false) {
 async function afterTests({ keychain, compass }) {
   try {
     if (compass) {
+      await printLogs(compass);
```

Comment:
If we want client logs for debugging I would suggest in addition to this method to use spectron api, get all the renderer logs with [`client.getRendererProcessLogs()`](https://github.com/electron-userland/spectron#clientgetrenderprocesslogs) and store them in a file, similar to what we are [already doing here](https://github.com/mongodb-js/compass/blob/342bee47b8416b049a6b67be4745e1aac4a41639/packages/compass-e2e-tests/helpers/compass.js#L164-L169). That way we will have some logs that are in chronological order instead of "by type"

Reply:
OK. Didn't know about getRendererProcessLogs. I don't mind changing it. I just want the error logs _somewhere_.

Reply:
Let me know what you think https://github.com/mongodb-js/compass/blob/0d579b1f31d4dcbe1815ad720911e5fbb52735d6/packages/compass-e2e-tests/helpers/compass.js#L161-L211